mappings/c4b65048fd8319a959690d59f66721d638865050cb90765b626309204265726368436f696e.json

```diff
@@ -1,30 +1,52 @@
+        "signatures": [
+            {
+                "signature": "b747b7366c61bc8148834224a08fc5183c256fbdb13432570bebd9dca2b7e3f3c265c594fb7cab369d0e91590c3082ddc552f8d897f7f67d7d5c2511e738b90a",
+                "publicKey": "c3d39acf2d48273bb13e88ff2aec16db8b013f1acea4ca1577dd8ddfc38bcd60"
+            },
+            {
+                "signature": "a214ea41c644ee5e2d5b709e0a9b86e8e4f7a82bfce097774d3a059e48901e39a2e68af35c3b3990c0d65e8c6ca880f46c56fdb1bc9cc2f112e9307acd418e01",
+                "publicKey": "7e71d327449e0af3c03952168017938e7a4605feaf62da81d33b0baad0fbfcc4"
+            },
+            {
+                "signature": "b747b7366c61bc8148834224a08fc5183c256fbdb13432570bebd9dca2b7e3f3c265c594fb7cab369d0e91590c3082ddc552f8d897f7f67d7d5c2511e738b90a",
```

Comment:
duplicate signatures in there - can you remove the extra ones?

Reply:
Duplicate sig removed

Reply:
it still shows up - holding on merging due to unexpected signatures / behavior of the CI checks